test/dialects/tsql_test.py

```diff
@@ -0,0 +1,16 @@
+"""Tests specific to the postgres dialect."""
+
+import pytest
+
+from sqlfluff.core import FluffConfig, Linter
+
+
+
+def test_space_is_not_reserved(raw):
+    """Ensure that SPACE is not treated as reserved."""
```

Comment:
This currently fails. You need to add this to `dialect_tsql` if you want to run this test:

```
tsql_dialect.sets("unreserved_keywords").remove("SPACE")
```

Maybe this test will pass when you actually load the TSQL Keywords?

To be honest I'd just delete this file for now, and add it later if needed.